### PR TITLE
Review fixes

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: src/Raven.Studio/typescript/common/queryCompleter.ts [hunk 1]
```diff
@@ -453,7 +453,10 @@ class queryCompleter {
         callback(null,  keywords.map(keyword  => {
             const word = <autoCompleteWordList>keyword;
             word.caption = _.trim(keyword.value, "'");
-            keyword.value += " ";
+            if (keyword.value.indexOf(" ") >= 0){
```
Comment (ml054) [inline src/Raven.Studio/typescript/common/queryCompleter.ts]: String has includes method :)

Comment (fitzchak) [reply src/Raven.Studio/typescript/common/queryCompleter.ts]: I expected it to have `Contains`... thanks, I'll update the code.

File: src/Raven.Studio/typescript/common/queryCompleter.ts [hunk 2]
```diff
@@ -453,7 +453,10 @@ class queryCompleter {
         callback(null,  keywords.map(keyword  => {
             const word = <autoCompleteWordList>keyword;
             word.caption = _.trim(keyword.value, "'");
-            keyword.value += " ";
+            if (keyword.value.includes(" ")){
```
Comment (ayende) [inline src/Raven.Studio/typescript/common/queryCompleter.ts:456]: Reverse the logic, if the collection name isn't just A-Z, 0-9 _, then we need qoutes.

Comment (fitzchak) [reply src/Raven.Studio/typescript/common/queryCompleter.ts:456]: What other values should be escaped? (other than eascape characters, which contains `\`)
Collection names in other languages are just working.

Comment (ayende) [reply src/Raven.Studio/typescript/common/queryCompleter.ts:456]: IsLetterOrDigit ?

Comment (fitzchak) [reply src/Raven.Studio/typescript/common/queryCompleter.ts:456]: Added to the todo list.

Comment (fitzchak) [reply src/Raven.Studio/typescript/common/queryCompleter.ts:456]: Done, currently only for English words but we might add more to the regex:
https://github.com/fitzchak/ravendb/blob/v4.0/src/Raven.Studio/typescript/common/queryCompleter.ts#L470

**Pull request merged**